calc/l3_route_resolver.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 
-	libapi "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	libapiv3 "github.com/projectcalico/libcalico-go/lib/apis/v3"
```

Comment:
I think this should be "apiv3", same as it was before in this file, so that no changes are needed after this point.